Framework/PythonInterface/mantid/_testing/AssertAlmostEqual.py

```diff
@@ -8,7 +8,7 @@
 from mantid.simpleapi import CompareWorkspaces
 
 
-def assert_almost_equal(Workspace1, Workspace2, rtol=0.0, atol=0.0):
+def assert_almost_equal(Workspace1, Workspace2, **kwargs):
```

Comment:
The original design was that after you `from mantid.api.AlgorithmProperty import EMPTY_DBL` this should be
```suggestion
def assert_almost_equal(Workspace1, Workspace2, rtol=EMPTY_DBL, atol=EMPTY_DBL, **kwargs):
```
which requires more changes below. Alternatively, the signature could be
```
def assert_almost_equal(Workspace1, Workspace2, rtol=None, atol=None, **kwargs):
``` 
which would aid people in how to properly use this.